nucypher/cli/painting.py

```diff
@@ -573,3 +573,31 @@ def paint_locked_tokens_status(emitter, agent, periods) -> None:
         emitter.echo(f"{bucket_range:>9}: {box_plot:60}"
                      f"Min: {NU.from_nunits(bucket_min)} - Max: {NU.from_nunits(bucket_max)}")
 
+
+def paint_input_allocation_file(emitter, allocations) -> None:
+    emitter.echo(f"\n{'='*46} STAGED ALLOCATIONS {'='*45}", bold=True)
```

Comment:
Include the total number of allocation to be performed here?

Reply:
✅